Components/Queues/QueueManager.php

```diff
@@ -147,7 +147,7 @@ public function send(string $queueName, string $message, array $headers = [])
         return $this->exchange->publish($message, $queueName, AMQP_NOPARAM, $headers);
     }
 
-    public function addConnection(\AMQPConnection $connection)
+    public function addConnection($connection)
```

Comment:
Is this QueueManager used on your implementation?

Reply:
Ok, $connection param is typed again.